src/helpers/__tests__/highlight-test.ts

```diff
@@ -3,7 +3,6 @@ import highlight from '../highlight';
 const NONE = 'none' as const;
 const FULL = 'full' as const;
 
-/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Can you explain which config allowed it not having this comment?

Reply:
the camelcase lint rule no longer exists